misk/src/main/kotlin/misk/tracing/backends/zipkin/ZipkinBackendConfig.kt

```diff
@@ -0,0 +1,8 @@
+package misk.tracing.backends.zipkin
+
+import misk.config.Config
+
+data class ZipkinBackendConfig(
+  val collector_url: String,
+  val reporter: ZipkinReporterConfig?
```

Comment:
Is this really optional?

Reply:
Yeah, Zipkin gives you sane defaults if not present.